### ISSUE TITLE
python_SNNDPC cannot find centoid of jain dataset correctly

### ISSUE THREAD

**Issue by CHENC-orld** [liurui39660/SnnDpc#6]
When I run the python source code, I cannot get the result as the paper show when clustering the jain dataset. Is anything wrong？ Wrong jain dataset or my python environment?

**Comment by liurui39660** [comment]
According to the old Matlab version source code, you need to manually choose the center for the Jain dataset. I can't remember if manual choosing is implemented in the python version. If not, please reply to me and I'll add this soon.

**Comment by CHENC-orld** [comment]
"manually choose the center" , do you mean the nc parameter? I have set the nc = 2, but it has the same result as dpc.

**Comment by liurui39660** [comment]
Nope, you should be able to manually choose the center from the $\rho-\delta$ graph (or whatever its name) by clicking on the graph. IIRC, for Jain, you'd choose the two points with the highest $\delta$, so the decision graph (the $\gamma$ one) may not be helpful in this case.

**Comment by CHENC-orld** [comment]
Thanks, I got it. There has no "manually choose the center" part, it automatically choose centers with highest $\rho*\delta$.

**Comment by liurui39660** [comment]
Apologize for the inconvenience. If you are in a hurry but have Matlab installed, I'd suggest you try the Matlab version. Otherwise, I'll see if I can add this tomorrow. But no guarantee, as those GUI things may require more effort.

**Comment by CHENC-orld** [comment]
Thanks a lot.
